tests/processes/ProcessingTest.py

```diff
@@ -507,19 +509,20 @@ def test_ignore_glob(self):
     def test_yield_ignore_ranges(self):
         test_file_dict_a = {'f':
                             ('# Ignore aBear\n',
-                             'a_string = "This string should be ignored"\n')}
+                             'a_string = "This string should not be '
+                             'tested"\n')}
         test_ignore_range_a = list(yield_ignore_ranges(test_file_dict_a))
         for test_bears, test_source_range in test_ignore_range_a:
             self.assertEqual(test_bears, ['abear'])
             self.assertEqual(test_source_range.start.line, 1)
             self.assertEqual(test_source_range.start.column, 1)
             self.assertEqual(test_source_range.end.line, 2)
-            self.assertEqual(test_source_range.end.column, 43)
+            self.assertEqual(test_source_range.end.column, 45)
 
         test_file_dict_b = {'f':
                             ('# start Ignoring bBear\n',
-                             'b_string = "This string should be ignored"\n',
-                             '# stop ignoring\n')}
+                             'b_string = "This string should not be '
+                             'tested"\n', '# stop ignoring\n')}
```

Comment:
Why are you changing ``ignored`` to ``not tested`` everywhere in the code?

Reply:
because ignore is now a reserved keyword and you cannot use it anywhere else except to ignore lines of code. If you really want to, then I have opened https://github.com/coala/coala/issues/5080